java-checks/src/main/java/org/sonar/java/checks/CatchUsesExceptionWithContextCheck.java

```diff
@@ -19,9 +19,11 @@
  */
 package org.sonar.java.checks;
 
-import com.google.common.base.Splitter;
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Lists;
+import java.util.ArrayDeque;
```

Comment:
Please don't change order of imports. These are useless changes. Did you used had a look at our coding style? https://github.com/SonarSource/sonar-developer-toolset#code-style